code/modules/surgery/organs/brain.dm

```diff
@@ -187,13 +187,14 @@
 /obj/item/organ/internal/brain/prepare_eat()
 	return // Too important to eat.
 
+// Hello I am from the ministry of rubber forehead aliens how are you
 /obj/item/organ/internal/brain/slime
 	name = "slime core"
 	desc = "A complex, organic knot of jelly and crystalline particles."
 	icon = 'icons/mob/slimes.dmi'
 	icon_state = "green slime extract"
 	mmi_icon_state = "slime_mmi"
-//	parent_organ = "chest" Hello I am from the ministry of rubber forehead aliens how are you
+	organ_datums = list(/datum/organ/heart, /datum/organ/lungs)
```

Comment:
you are not using the defines from earlier?

Reply:
Not sure if that'll work as intended, seeing the defines are
```
#define ORGAN_DATUM_HEART	"heart"
#define ORGAN_DATUM_LUNGS	"lungs"
```
While these are the actual datums that need to be added

Reply:
oh